.github/workflows/build-tests.yml

```diff
@@ -21,9 +21,18 @@ jobs:
 
     - name: Install dependencies
       run: |
-        dnf install -y dnf-plugins-core maven rpm-build
-        dnf copr enable -y ${{ needs.init.outputs.repo }}
-        dnf builddep -y --spec tomcatjss.spec
+        dnf install -y dnf-plugins-core moby-engine maven rpm-build
+        if [ -n "$COPR_REPO" ]; then dnf copr enable -y $COPR_REPO; fi
+        dnf builddep -y --skip-unavailable --spec tomcatjss.spec
+      env:
+        COPR_REPO: ${{ needs.init.outputs.repo }}
+
+    - name: Install JSS packages from jss-builder
+      run: |
+        docker pull ghcr.io/dogtagpki/jss-builder
+        docker create --name=jss-builder ghcr.io/dogtagpki/jss-builder
+        docker cp jss-builder:/root/jss/build/RPMS /tmp/RPMS/
+        dnf localinstall -y /tmp/RPMS/*
```

Comment:
When this is executed on the master it is OK but when this is in a branch I think we have problems. I do not see version in jss-builder so I suppose it is from the master but we need the matching version with the branch we are using. Additionally, if we provide a COPR_REPO for the branches then the jss-builder should not be used otherwise because it is not from the right version. We could go ahead with this and solve the branch problems later, when they are evident.

Reply:
Yes, the code will use the `latest` images by default. I've updated the PR to publish and use the `latest` images explicitly. In other branches we will need to replace the `latest` with the right versions for the branch. 

Later we can figure out how to centralize the version numbers so that we can change it in a single place, but it looks like the `COPY --from=<source image>` command in the `Dockerfile` doesn't accept any variable in the source image, so we might need to hard-code the version number there.

Reply:
I think we can replace the `latest` with the branch name in the future but need some investigation we can work on that with separate PR  when a new branch is created from the master. For now we can merge this

Reply:
Yeah, the `latest` doesn't necessarily have to be mapped to `master` branch. Ideally it should point to the latest official release.